{{cookiecutter.project_slug}}/clients/web/vue3/src/views/ResetPassword.vue

```diff
@@ -24,18 +25,21 @@
           type="password"
           label="Confirm Password"
           placeholder="Confirm Password"
+          :id="form.confirmPassword.id"
+          data-cy=""
```

Comment:
this `data-cy` either needs a value for a test to pass....or it's not needed at all